plugins/apitracing/src/lib/CMakeLists.txt

```
@@ -41,6 +41,15 @@ FetchContent_MakeAvailable(yaml-cpp)
 set_property(TARGET yaml-cpp PROPERTY POSITION_INDEPENDENT_CODE TRUE)
 target_link_libraries(apitracing-obj PUBLIC yaml-cpp)
 
+# Setup json-cpp
+FetchContent_Declare(
+        json-cpp
+        GIT_REPOSITORY https://github.com/open-source-parsers/jsoncpp.git
+        GIT_TAG 1.9.5
+)
+FetchContent_MakeAvailable(json-cpp)
+target_link_libraries(apitracing-obj PUBLIC jsoncpp_lib)
```

```suggestion
target_link_libraries(apitracing-obj PUBLIC jsoncpp_static)
```
You may need to enable the static build. Perhaps you should also disable the shared build in order to reducde build times.

---

Should already be disabled by:
option(YAML_BUILD_SHARED_LIBS "" OFF)